SingularityUI/app/views/requestForm.jsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+import View from './view';
+
+import { Provider } from 'react-redux';
+
+import RequestFormPage from '../components/requestForm/RequestForm'
+
+
+class RequestFormView extends View {
+  initialize({store, editing}) {
+    this.store = store;
+    this.editing = editing;
```

Comment:
There is no logic to unload the request. So if you edit a request, then try to create a new one,  the form will think you're still editing the first one and render an edit form.
The other option is to make another reducer to explicitly unload the request (which is fine if you think it would be better, but it is a bunch of extra logic).